tests/unit/rules/javascript/test.deprecated_browser_apis.js

```diff
@@ -1,19 +1,36 @@
+/* eslint-disable import/namespace */
```

Comment:
This is going to disable the import/namespace rule for the entire file, which may be a bit extreme, not a super big deal given that this is a test case, but isn't it enough to only disable this rule at line 24?